src/diffpy/morph/morphapp.py

```diff
@@ -419,9 +432,9 @@ def custom_error(self, msg):
             "using a serial file NAMESFILE. The format of NAMESFILE should be "
             "as follows: each target PDF is an entry in NAMESFILE. For each "
             "entry, there should be a key {__save_morph_as__} whose value "
-            "specifies the name to save the manipulated PDF as. An example "
```

Comment:
another issue to make as claning before release (not now) would be to do a global search for PDF and root it out wherever it survives.   Like a weed......